ova/generate_ova.sh

```diff
@@ -55,163 +64,191 @@ clean() {
 
     cd ${scriptpath}
     vagrant destroy -f
-    rm -f ${OVA_VM} ${OVF_VM} ${OVA_VMDK} ${OVA_FIXED} Config_files/filebeat.yml
-
+    OVA_VMDK="wazuh-${OVA_VERSION}-disk001.vmdk"
+    rm -f ${OVA_VM} ${OVF_VM} ${OVA_VMDK} ${OVA_FIXED}
+    
     exit ${exit_code}
 }
 
 build_ova() {
 
+    # Files
     OVA_VM="wazuh-${OVA_VERSION}.ova"
     OVF_VM="wazuh-${OVA_VERSION}.ovf"
     OVA_FIXED="wazuh-${OVA_VERSION}-fixed.ova"
-    OVA_VMDK="wazuh-${OVA_VERSION}-disk001.vmdk"
-    ELK_MAJOR=`echo ${ELK_VERSION} | cut -d"." -f1`
-    export BRANCH
-
-    if [ -e "${OVA_VM}" ] || [ -e "${OVA_VM}" ]; then
-        rm -f ${OVA_VM} ${OVF_VM}
-    fi
+    export PACKAGES_REPOSITORY
 
-    #Download filebeat.yml and enable geoip
-    if [ ${ELK_MAJOR} -eq 7 ]; then
-        curl -so Config_files/filebeat.yml https://raw.githubusercontent.com/wazuh/wazuh/v${WAZUH_VERSION}/extensions/filebeat/7.x/filebeat.yml
-        sed -i "s|#pipeline: geoip|pipeline: geoip|" Config_files/filebeat.yml
+    # Delete OVA/OVF files if exists
+    if [ -e "${OUTPUT_DIR}/${OVA_VM}" ] || [ -e "${OUTPUT_DIR}/${OVF_VM}" ]; then
+        rm -f ${OUTPUT_DIR}/${OVA_VM} ${OUTPUT_DIR}/${OVF_VM}
     fi
 
-
-    # Vagrant will provision the VM with all the software. (See vagrant file)
+    # Vagrant will provision the VM with all the software. (See vagrantfile)
     vagrant destroy -f
     vagrant up || clean 1
     vagrant suspend
-    VM_EXPORT=$(vboxmanage list vms | grep -i vm_wazuh | cut -d "\"" -f2)
-    # OVA creation with all metadata information.
-    vboxmanage export ${VM_EXPORT} -o ${OVA_VM} --vsys 0 --product "Wazuh v${WAZUH_VERSION} OVA" --producturl "https://packages.wazuh.com/vm/wazuh-${OVA_VERSION}.ova" --vendor "Wazuh, inc <info@wazuh.com>" --vendorurl "https://wazuh.com" --version "$OVA_VERSION" --description "Wazuh helps you to gain security visibility into your infrastructure by monitoring hosts at an operating system and application level. It provides the following capabilities: log analysis, file integrity monitoring, intrusions detection and policy and compliance monitoring." || clean 1
 
+    echo "Exporting ova"
+
+    # Get machine name
+    VM_EXPORT=$(vboxmanage list vms | grep -i vm_wazuh | cut -d "\"" -f2)
+    
+    # Create OVA with machine
+    vboxmanage export ${VM_EXPORT} -o ${OVA_VM} \
+    --vsys 0 \
+    --product "Wazuh v${WAZUH_VERSION} OVA" \
+    --producturl "https://packages.wazuh.com/vm/wazuh-${OVA_VERSION}.ova" \
+    --vendor "Wazuh, inc <info@wazuh.com>" --vendorurl "https://wazuh.com" \
+    --version "$OVA_VERSION" --description "Wazuh helps you to gain security visibility into your infrastructure by monitoring hosts at an operating system and application level. It provides the following capabilities: log analysis, file integrity monitoring, intrusions detection and policy and compliance monitoring." \
+    || clean 1
+
+    # Destroy vagrant machine
```

Comment:
Over-commenting

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/746/commits/27e2d14fcdf50546e45b5066ded7e2863482c4d4